src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +732,75 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export')
+def computer_export():
+    """Export the setup or configuration of a computer."""
+
+
+@computer_export.command('setup')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False, path_type=pathlib.Path))
+@with_dbenv()
+def computer_export_setup(computer, output_file):
+    """Export computer setup to a yaml file."""
+    import yaml
+
+    computer_setup = {
+        'label': computer.label,
+        'description': computer.description,
+        'hostname': computer.hostname,
+        'transport': computer.transport_type,
+        'scheduler': computer.scheduler_type,
+        'work_dir': computer.get_workdir(),
+        'shebang': computer.get_shebang(),
+        'mpirun_command': ' '.join(computer.get_mpirun_command()),
+        'mpiprocs_per_machine': computer.get_default_mpiprocs_per_machine(),
+        'default_memory_per_machine': computer.get_default_memory_per_machine(),
+        'prepend_text': computer.get_prepend_text(),
+        'append_text': computer.get_append_text(),
+        'use_double_quotes': computer.get_use_double_quotes(),
+    }
+    try:
+        output_file.write_text(yaml.dump(computer_setup, sort_keys=True), 'utf-8')
+        echo.echo_success(f"Computer<{computer.pk}> {computer.label} setup exported to file '{output_file}'.")
```

Comment:
I think this should rather go in the `else`-block of the try/except. Otherwise, if there would be a bug hiding in this line that raises, it would be caught by the exception that is designed to catch errors in the actual dumping

Reply:
done